integration/approaches/topDownVsBottomUp/text.md

```diff
@@ -1,4 +1,4 @@
-<span id="title">Top-down VS bottom-up integration</span>
+<span id="title">Top-down vs Bottom-up integration</span>
```

Comment:
Is there a special rule for this case? Seems wikipedia used `bottom-up` https://en.wikipedia.org/wiki/Top-down_and_bottom-up_design

Reply:
Ah I capitalized it to separate it as two elements of the `vs` and keep it consistent with the remaining headings containing `vs`.

Reply:
You came up with that convention or is there some precedence from elsewhere? Is it mentioned in Google style guide? Otherwise we should go with the general rule rather than create a special case.

Reply:
I had come up with the convention on my own to make it consistent across. But now I have found an example, so changing all accordingly.

Reply:
@damithc Google documentation has used 'versus` instead of vs in heading at one instance. Therefore, I have followed that.